networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py

```diff
@@ -161,9 +153,12 @@ def _realize(self, resource_type, delete, convertor, os_o, provider_o):
         meta = self.metadata(resource_type, os_id)
         if meta:
             if delete:
+                LOG.info(report, "deleted")
```

Comment:
Should that really stay in? It looks like debug code to me. Honest question, I do not know

Reply:
I will change the priority.